server/aws-lsp-codewhisperer/src/language-server/netTransform/artifactManager.ts

```diff
@@ -144,6 +144,25 @@ export class ArtifactManager {
         }
         this.logging.log('Total project references: ' + projects.length)
 
+        let packages: string[] = []
+        if (request.PackageReferences != null) {
+            for (const pkg of request.PackageReferences) {
+                if (pkg.NetCompatiblePackageFilePath == null) {
+                    continue
+                }
```

Comment:
Do we want to guard against empty strings as well?

Reply:
also NetCompatiblePackageFilePath field may not exist, depend on toolkits or lsp released first, better to check the case